### DISCUSSION TITLE
v0.0.0 尘降 🌟

### DISCUSSION THREAD

**Discussion by A-kirami** [A-kirami/KiramiBot#1] [Announcements]
从22年初立项，到23年的今天，期间又是磨蹭又是摸鱼，现在终于发布了真是可喜可贺。

目前还缺少相关生态基础设施，关于 [KiramiBot](https://github.com/A-kirami/KiramiBot) 的未来发展请查看[路线图](https://github.com/users/A-kirami/projects/6)。

<hr /><em>This discussion was created from the release <a href='https://github.com/A-kirami/KiramiBot/releases/tag/v0.0.0'>v0.0.0 尘降 🌟</a>.</em>

**Comment by Chr0n0stasis** [comment]
路线图 指404网页

**Reply by A-kirami** [reply]
忘记公开了，已经好了
